maxmind_connector.py

```diff
@@ -226,7 +226,7 @@ def _should_download_new_db(self):
         self.debug_print('Checking if the current database is up to date.')
         db_url = DB_DOWNLOAD_URL.format(self._license_key)
 
-        r = requests.head(db_url)
+        r = requests.head(db_url, timeout=DEFAULT_REQUEST_TIMEOUT)
```

Comment:
Timeout is added to fix the SemGrep piepline.

Reply:
Fix is recommended by this log https://github.com/splunk-soar-connectors/maxmind/runs/4612649991?check_suite_focus=true